RoslynPluginGenerator/AnalyzerPluginGenerator.cs

```diff
@@ -118,28 +189,31 @@ public bool Generate(ProcessedArgs args)
                 createdJarFilePath = BuildPlugin(definition, args.OutputDirectory);
             }
 
-            LogSummary(createdJarFilePath, generatedSqaleFile, licenseAcceptancePackages);
+            LogSummary(createdJarFilePath, generatedSqaleFile);
 
             return createdJarFilePath != null;
         }
 
-        private void LogSummary(string createdJarFilePath, string generatedSqaleFile, IEnumerable<IPackage> licenseAcceptancePackages)
+        private void LogSummary(string createdJarFilePath, string generatedSqaleFile)
```

Comment:
It is, but only because it appears at the the end of _every_ plugin generation. This should be changed.

This links into the SQALE output comment, so I'll think how about how to proceed. Maybe we could output one SQALE warning at the end, no matter how many plugins lacked it. That will mean changing the output message a bit.